src/omero_zarr/util.py

```diff
@@ -18,3 +20,16 @@ def print_status(t0: int, t: int, count: int, total: int) -> None:
         eta = "NA"
     status = f"{percent_done:.2f}% done, ETA: {eta}"
     print(status, end="\r", flush=True)
+
+
+def open_store(name: str) -> FSStore:
+    """
+    Create an FSStore instance that supports nested storage of chunks.
+    """
+    return FSStore(
```

Comment:
I assume this logic could be delegated to `ome_zarr.format` down the line? This would also make the format version used by this plugin more explicit.

Immediate caveat looking at https://github.com/ome/ome-zarr-py/blob/62b49da4b7200384dac8dec8d9fce48bd727a967/ome_zarr/format.py#L101-L125 is that `normalize_keys` would need to configurable (unless it could be set to `True` here?).

Reply:
It was set to `False` because otherwise path with 'A/1` was normalised to `a/1`: https://github.com/ome/omero-cli-zarr/pull/59#discussion_r604823386

Reply:
That may have been because upstream was doing something wonky. Again, I _think_ that's all settled:

https://github.com/zarr-developers/zarr-python/pull/755

Reply:
If I want this repo to depend on that fix `zarr>=2.8.3` I'd either need to update that in ome-zarr-py (currently `"zarr>=2.8.1"`) or add `zarr` as a dependency to this repo?

Reply:
Happy for it to happen in either as long as we eventually unify. fwiw, zarr is now at 2.10.*